butane_core/src/migrations/adb.rs

```diff
@@ -113,6 +113,12 @@ impl TypeResolver {
         }
     }
     fn find_type(&self, key: &TypeKey) -> Option<TypeIdentifier> {
+        #[cfg(feature = "json")]
+        if let TypeKey::CustomType(ct) = key {
+            if ct.starts_with("HashMap < String,") {
```

Comment:
This is a horrible hack to avoid needing `butane_type`, which would require a `newtype` for each type of HashMap.

Reply:
Yeah, this is another great example of the limitations of trying to create the schema representation at compile time with only the AST and therefore why I'd like to move it to runtime.

Reply:
Since I haven't done that yet though, I'm reluctantly OK with the hack for now